### PR TITLE
add kube-fzf fish food

### LINKED ISSUE #1: Add fish food for kube-fzf
I'll raise a PR for this

### PR DESCRIPTION
**Pull request by karuppiah7890**
Closes #1

### CODE REVIEW

File: Food/kube-fzf.lua [hunk 1]
```diff
@@ -0,0 +1,127 @@
+local name = "kube-fzf"
+local version = "1.6.1"
+
+food = {
```
Comment (jskswamy) [inline Food/kube-fzf.lua:41]: is it possible to pull out packages for different architecture to a common method?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Fish Food
```
Comment (jskswamy) [inline README.md:1]: @karuppiah7890 could you add instruction on how to add this to fish?

**Comment by karuppiah7890** [comment]
^ @jskswamy

**Comment by karuppiah7890** [comment]
@jskswamy I have made the changes and pushed now 👍

**Pull request closed without merging**